osi_sensorviewconfiguration.proto

```diff
@@ -201,10 +201,15 @@ message SensorViewConfiguration
     // Omit Static Information
     //
     // This flag specifies whether \c GroundTruth information that
-    // was already provided using a GroundTruthInit parameter
+    // was already provided using a
+    // <a href="https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/sensor-model/spec/ground_truth_init_parameters.html">OMSP GroundTruthInit</a> parameter
```

Comment:
It should be OSMP instead of OMSP. I hope the spellcheck will go through then.

Reply:
Done